pkgs/top-level/all-packages.nix

```diff
@@ -7790,15 +7790,17 @@ with pkgs;
 
   phantomjs2 = libsForQt5.callPackage ../development/tools/phantomjs2 { };
 
-  pmccabe = callPackage ../development/tools/misc/pmccabe { };
-
   pkgconfig = callPackage ../development/tools/misc/pkgconfig {
     fetchurl = fetchurlBoot;
   };
   pkgconfigUpstream = lowPrio (pkgconfig.override { vanilla = true; });
 
+  pmccabe = callPackage ../development/tools/misc/pmccabe { };
```

Comment:
if you want to move this can you make it a separate commit?

Reply:
I think there is no need to move this, just leave the existing order or move the postman attribute.

Reply:
Yeah I did that to fixth e alphabetical ordering, but it's not related to the PR. I will remove that change.